src/documents/FetchResources.md

```diff
@@ -153,13 +132,13 @@ This shows a more complex example which checks for errors from the fetch process
 ```javascript
 var myResourceSpecs = {
     template1: {
-        href: "html/template1.html"
+        url: "html/template1.html"
```

Comment:
The example that comes before this has a myResourceSpecs object that uses `href` instead of `url`. Should the other example also be updated, or is there a reason that this particular case has been changed?

Reply:
Thanks for spotting this, yes the use of href was deprecated as of Infusion 3.0 and has been removed for Infusion 4.0. I'll add this to our deprecation notes.